code/modules/paperwork/pen.dm

```diff
@@ -196,6 +196,13 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut") //these wont show up if the pen is off
 	var/on = FALSE
 
+/obj/item/pen/edagger/suicide_act(mob/living/user)
+	if(on)
+		user.visible_message("<span class='suicide'>[user] jabs [src] into their neck! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
you can cancel the suicide act so this should be "is trying to jab" or else you could continuously jab yourself with the edagger :^)

Reply:
suicide false is for items so horrible they shouldn't even be in the game, like SORD

Reply:
Use is jabbing then

Reply:
there is no suicide false here?